devel/tbb/Portfile

```diff
@@ -35,6 +31,9 @@ use_configure       no
 # Force a compatible compiler
 compiler.blacklist-append *gcc* {clang < 602} {macports-clang-3.[0-7]}
 
+# install it into libexec to avoid conflict with onetbb port
+prefix              ${prefix}/libexec/${name}
```

Comment:
AFAIK, we typically don't change `prefix` within a port, as that could cause unexpected issues.

Instead, it's better to define your own variable, representing the install prefix for your port.

Anyone else... thoughts?